addons/account/models/account_move.py

```diff
@@ -73,6 +73,10 @@ def _sequence_monthly_regex(self):
     def _sequence_yearly_regex(self):
         return self.journal_id.sequence_override_regex or super()._sequence_yearly_regex
 
+    @property
+    def _sequence_year_range_regex(self):
```

Comment:
Note: the same will be needed for `_sequence_year_range_monthly_regex` in 18.0, I (or whoever approves this PR) will take care of it in the forward port